socorro/schemas/README.md

```diff
@@ -1,5 +1,46 @@
 # Socorro JSON Schemas
 
-This folder contains JSON Schema files describing the documents that Socorro generates. At the moment only the Processed Crash (the one generated by Socorro's processors) is described.
+This folder contains JSON Schema files describing the documents that Socorro
+generates. At the moment only the Processed Crash (the one generated by
+Socorro's processors) is described.
 
-These files will be used as a "contract" between Socorro and other systems to which we might send our data.
+These files will be used as a "contract" between Socorro and other systems
+to which we might send our data.
+
+## Changing The Schema
+
+You can change the schema but if you do, **remember to increment the
+`$target_version`**.
+
+The JSON Schema should contain a key called `$target_version`.
+
+* It's important this is an integer that goes up
+* *Don't change this* if you're...
+  * Adding more keys
+  * Editing comments (content of `description` values)
+* *Do change this* if you're...
+  * Changing the type definition of an *existing* key in any way
+  * Add or remove keys from a `required` sub-key
+
+Yes, if you add more keys, don't change the version.
+For example, if you want to add...:
+
+```json
+"addons_checksum": {
+    "type": ["string", "null"],
+    "description": "Sample specimen"
+}
+```
+...to the root or inside an existing key. Then **don't** change the version.
+
+However, if you change the type definition, E.g.
+
+```diff
+"addons_checksum": {
+-   "type": ["string", "null"],
++   "type": ["integer", "null"],
+    "description": "Sample specimen"
+},
+```
+Then you have to change the `$target_version` number to `$target_version+1`
+whatever that new number might be.
```

Comment:
I think you can simplify that by saying ``Then you have to increment the `$target_version` number. ``.